packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2081,7 +2081,12 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
 
     if (value.text == _value.text && value.composing == _value.composing) {
       // `selection` is the only change.
-      _handleSelectionChanged(value.selection, (_textInputConnection?.scribbleInProgress ?? false) ? SelectionChangedCause.scribble : SelectionChangedCause.keyboard);
+      _handleSelectionChanged(
+        value.selection,
+        Scribble.scribbleInProgress
```

Comment:
The change makes the line a bit different. Scribble could be in progress while there's no active connection.

Reply:
Is that handled by the part above that was added for autofill?

```dart
    if (!_shouldCreateInputConnection) {
      return;
    }
```

Reply:
The change can still come from the keyboard when scribble is in progress (especially when there's a connected hardware keyboard). But since it's already there I'm fine with keeping this as is.

Reply:
Sounds good, let's keep an eye out if any issues are filed about typing with a keyboard while doing handwriting.